js/app/modules/selection/allHighlights.js

```diff
@@ -18,11 +18,23 @@ const AllHighlights = new Module.Class({
     },
 
     construct_query_object: function (limit, query_index) {
-        if (query_index > 0)
-            return null;
+        let tags_filter = ['EknArticleObject'];
+        let excluded_tags = ['EknFeaturedTag'];
+        switch (query_index) {
```

Comment:
Yeah I can do that - I mostly took a look at another selection filter with multiple queries for stylistic reference to keep consistency and it used the switch statement: https://github.com/endlessm/eos-knowledge-lib/blob/master/js/app/modules/selection/supplementaryArticles.js#L31-L39